lib/licensee/content_helper.rb

```diff
@@ -4,13 +4,14 @@
 module Licensee
   module ContentHelper
     DIGEST = Digest::SHA1
-    END_OF_TERMS_REGEX = /^\s*end of terms and conditions\s*$/i
+    END_OF_TERMS_REGEX = /^[\s#]*end of terms and conditions\s*$/i
     HR_REGEX = /[=\-\*][=\-\*\s]{3,}/
     ALT_TITLE_REGEX = License::ALT_TITLE_REGEX
     ALL_RIGHTS_RESERVED_REGEX = /\Aall rights reserved\.?$/i
     WHITESPACE_REGEX = /\s+/
     MARKDOWN_HEADING_REGEX = /\A\s*#+/
     VERSION_REGEX = /\Aversion.*$/i
+    MARKUP_REGEX = /[^\w'\.\-,]+/
```

Comment:
Is the `'` there because it can make a word possessive, but not `"` because it can't change meaning?

If a comma is considered meaningful, shouldn't a semicolon also be? What about parens?

Reply:
You're absolutely right (and that was my original thinking). I checked the SPDX license matching spec, and it says formatting should be ignored, but punctuation should not. Updated with a blacklist of Markdown markup characters via 08f3779.